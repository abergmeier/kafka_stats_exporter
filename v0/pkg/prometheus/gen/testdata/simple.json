{
    "name": "rdkafka#producer-1",
    "brokers": {
      "localhost:9092/2": {
        "name": "localhost:9092/2",
        "rxbytes": 15708
      }
    },
    "rx_bytes": 31084
  }
