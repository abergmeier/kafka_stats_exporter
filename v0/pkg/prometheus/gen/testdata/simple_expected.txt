# HELP brokers_rxbytes_total Total number of bytes received
# TYPE brokers_rxbytes_total counter
brokers_rxbytes_total{brokers_name="localhost:9092/2",name="rdkafka#producer-1"} 15708

# HELP rx_bytes_total Total number of bytes received from Kafka brokers
# TYPE rx_bytes_total counter
rx_bytes_total{name="rdkafka#producer-1"} 31084
